server/app/assets/javascripts/map.ts

```diff
@@ -0,0 +1,60 @@
+import {Map} from 'maplibre-gl'
+
+export const init = () => {
+  const mapElements = document.querySelectorAll('[id^="cf-map-"]')
+
+  if (mapElements.length === 0) return
```

Comment:
(note) Always use braces, even for one liner `if`s.

(blocking) Remove the if check. It's redundant as nothing will happen in the foreach if there are no elements.